apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PriorityList.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Google.Cloud.Spanner.V1
+{
+    /// <summary>
+    /// A simple sorted list that allows items to return equality for IComparable.CompareTo
```

Comment:
I think it would be worth clarifying what's mean by "allows" here - is it really meaning this is acting as a SortedSet? It looks like that's what you're doing in Add, but an earlier comment around equal hash codes suggests something different.